dashboard/app/controllers/musiclab_controller.rb

```diff
@@ -1,7 +1,9 @@
 class MusiclabController < ApplicationController
   ANALYTICS_KEY = CDO.amplitude_api_key
   NUM_MINI_PLAYER_PROJECTS = 5
-  # Hard-coded list of channel IDs for the mini player if DCDO flag is set to `true`
+  # The hard-coded list of channel IDs for the mini player if the 'get_channel_ids_from_featured_projects_gallery'
+  # DCDO flag is set to `false`. The default for the DCDO flag is `true` so that the channel IDs are retrieved
+  # from the active music lab featured projects by default. See lines 64-69 below.
```

Comment:
nit: second sentence could be something like
```If the DCDO flag is `true`, the channel IDs are randomly selected from the set of active music lab featured projects. See get_selected_channel_ids below.``` 
in case the defaults/line numbers happen to change?